meilisearch_interface.go

```diff
@@ -27,6 +28,9 @@ type ServiceManager interface {
 	WebhookManager() WebhookManager
 	WebhookReader() WebhookReader
 
+	NetworkManager() NetworksManager
```

Comment:
I think it's better not to separate network methods; we can put them in the service manager and reader.